diff --git a/Configurations/windows-makefile.tmpl b/Configurations/windows-makefile.tmpl
index b587212..c031cfb 100644
--- a/Configurations/windows-makefile.tmpl
+++ b/Configurations/windows-makefile.tmpl
@@ -546,6 +546,7 @@ install_dev: install_runtime_libs build_inst_dev
 	@"$(PERL)" "$(SRCDIR)\util\mkdir-p.pl" "$(libdir)"
 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_LIBS) "$(libdir)"
 	@if "$(SHLIBS)"=="" \
+	 @if "$(INSTALL_PDBS)"=="ON" \
 	 "$(PERL)" "$(SRCDIR)\util\copy.pl" ossl_static.pdb "$(libdir)"
 	@"$(PERL)" "$(SRCDIR)\util\mkdir-p.pl" "$(CMAKECONFIGDIR)"
 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_EXPORTERS_CMAKE) "$(CMAKECONFIGDIR)"
@@ -559,6 +560,7 @@ install_modules: _install_modules_deps
 	@if not "$(INSTALL_MODULES)"=="" \
 	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_MODULES) "$(MODULESDIR)"
 	@if not "$(INSTALL_MODULES)"=="" \
+	 @if "$(INSTALL_PDBS)"=="ON" \
 	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_MODULEPDBS) "$(MODULESDIR)"
 
 uninstall_modules:
@@ -572,6 +574,7 @@ install_runtime_libs: build_libs
 	@if not "$(SHLIBS)"=="" \
 	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_SHLIBS) "$(INSTALLTOP)\bin"
 	@if not "$(SHLIBS)"=="" \
+	 @if "$(INSTALL_PDBS)"=="ON" \
 	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_SHLIBPDBS) \
                                         "$(INSTALLTOP)\bin"
 
@@ -583,6 +586,7 @@ install_programs: install_runtime_libs build_inst_programs
 	@if not "$(INSTALL_PROGRAMS)"=="" \
 	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_PROGRAMS) \
                                         "$(INSTALLTOP)\bin"
-	@if not "$(INSTALL_PROGRAMPDBS)"=="" \
+	@if not "$(INSTALL_PROGRAMPDBS)"=="" \
+	 @if "$(INSTALL_PDBS)"=="ON" \
 	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_PROGRAMPDBS) \
                                         "$(INSTALLTOP)\bin"
