diff --git a/Configurations/windows-makefile.tmpl b/Configurations/windows-makefile.tmpl
index b587212..f472851 100644
--- a/Configurations/windows-makefile.tmpl
+++ b/Configurations/windows-makefile.tmpl
@@ -201,7 +201,7 @@ OPENSSLDIR_dev={- #
                           (file_name_is_absolute($config{openssldir}) ?
                                canonpath($config{openssldir})
                                : catdir($prefix, $config{openssldir}))
-                          : canonpath("$win_commonroot\\SSL");
+                          : canonpath("$win_commonroot\\OpenSSL\\SSL");
                   our ($openssldir_dev, $openssldir_dir, $openssldir_file) =
                       splitpath($openssldir, 1);
                   $openssldir_dev -}
@@ -218,7 +218,7 @@ MODULESDIR_dev={- use File::Spec::Functions qw(:DEFAULT splitpath catpath);
                       splitpath($modulesprefix, 1);
                   our $modulesdir_dev = $modulesprefix_dev;
                   our $modulesdir_dir =
-                      catdir($modulesprefix_dir, "ossl-modules");
+                      catdir($modulesprefix_dir, "../bin");
                   our $modulesdir = catpath($modulesdir_dev, $modulesdir_dir);
                   $modulesdir_dev -}
 MODULESDIR_dir={- canonpath($modulesdir_dir) -}
